test/primitives/base-element-test.js

```diff
@@ -84,11 +84,11 @@ describe('BaseElement', function() {
     });
 
     it('returns true when they are equal', function() {
-      expect(el.id.equals('foobar')).to.be.true;
```

Comment:
So, basically, we have decided that `el.id` giving the actual value is more common than this, right?